diff --git a/node_modules/pdfjs-dist/build/pdf.js b/node_modules/pdfjs-dist/build/pdf.js
index 4acf16b..198bdda 100644
--- a/node_modules/pdfjs-dist/build/pdf.js
+++ b/node_modules/pdfjs-dist/build/pdf.js
@@ -6244,7 +6244,7 @@ class NodeFilterFactory extends _base_factory.BaseFilterFactory {}
 exports.NodeFilterFactory = NodeFilterFactory;
 class NodeCanvasFactory extends _base_factory.BaseCanvasFactory {
   _createCanvas(width, height) {
-    const Canvas = require("canvas");
+    const Canvas = window.canvas || HTMLCanvasElement;
     return Canvas.createCanvas(width, height);
   }
 }
diff --git a/node_modules/pdfjs-dist/legacy/build/pdf.js b/node_modules/pdfjs-dist/legacy/build/pdf.js
index 512540d..0b741c6 100644
--- a/node_modules/pdfjs-dist/legacy/build/pdf.js
+++ b/node_modules/pdfjs-dist/legacy/build/pdf.js
@@ -10790,7 +10790,7 @@ var _util = __w_pdfjs_require__(1);
       return;
     }
     try {
-      globalThis.DOMMatrix = require("canvas").DOMMatrix;
+      globalThis.DOMMatrix = window.canvas.domMatrix || HTMLCanvasElement.domMatrix;
     } catch (ex) {
       (0, _util.warn)(`Cannot polyfill \`DOMMatrix\`, rendering may be broken: "${ex}".`);
     }
@@ -10802,7 +10802,7 @@ var _util = __w_pdfjs_require__(1);
     try {
       const {
         CanvasRenderingContext2D
-      } = require("canvas");
+      } = window.canvas || HTMLCanvasElement;
       const {
         polyfillPath2D
       } = require("path2d-polyfill");
@@ -10829,7 +10829,7 @@ class NodeFilterFactory extends _base_factory.BaseFilterFactory {}
 exports.NodeFilterFactory = NodeFilterFactory;
 class NodeCanvasFactory extends _base_factory.BaseCanvasFactory {
   _createCanvas(width, height) {
-    const Canvas = require("canvas");
+    const Canvas = window.canvas || HTMLCanvasElement;
     return Canvas.createCanvas(width, height);
   }
 }
